packages/@lwc/engine-server/src/renderer.ts

```diff
@@ -173,14 +173,22 @@ export const renderer: Renderer<HostNode, HostElement> = {
         return new Proxy(
             {},
             {
-                get(target, property) {
+                get(target, property): string {
+                    let value: string;
+
                     const styleAttribute = getStyleAttribute();
```

Comment:
_Not a change made in this pr, but may be you can take care of it here_
`getStyleAttribute()` has this `element.attributes.find`. element.attributes returns a NamedNodeMap. NamedNodeMap does not have a `find` method on it.

Reply:
@ravijayaramappa We're actually interacting with our DOM polyfill here